data_dir/cs_lwt_perf_big.yaml

```diff
@@ -0,0 +1,175 @@
+        cl3 smallint,
+        cl4 bigint,
+        cl5 double,
+        cl6 float,
+        cl7 tinyint,
+        cl8 decimal,
+        cl8 bigint,
+        cl10 text,
+        aux1 timeuuid,
+        aux2 int,
+        aux3 smallint,
+        aux4 bigint,
+        aux5 double,
+        aux6 float,
+        aux7 tinyint,
+        aux8 decimal,
+        aux9 bigint,
+        aux10 text,
+        PRIMARY KEY((pr1, pr2, pr3, pr4, pr5, pr6, pr7, pr8, pr9, pr10), cl1, cl2, cl3, cl4, cl5, cl6, cl7, cl8, cl9, cl10)
+  ) WITH compaction = { 'class':'LeveledCompactionStrategy' }
+    AND comment='A test table for cs_lwt_perf_big'
+
+### Column Distribution Specifications ###
+
+columnspec:
+  - name: pr1
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr2
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr3
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr4
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr5
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr6
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr7
+    size: uniform(1..254)
+    population: uniform(1..254)
+  - name: pr8
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr9
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: pr10
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl1
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl2
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl3
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl4
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl5
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl6
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl7
+    size: uniform(1..254)
+    population: uniform(1..254)
+  - name: cl8
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl9
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: cl10
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux1
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux2
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux3
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux4
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux5
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux6
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux7
+    size: uniform(1..254)
+    population: uniform(1..254)
+  - name: aux8
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux9
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+  - name: aux10
+    size: uniform(1..1k)
+    population: uniform(1..1k)
+### Batch Ratio Distribution Specifications ###
+
+insert:
+  partitions: fixed(256)
```

Comment:
With multiple clustering components this could generate very large batches (65k). I think optimal maximal batch size for this kind of test is ~20 statements, optimal median being even lower - 2-3 statements. Otherwise we're stress testing a very specific case, not too widespread in production (bulk load with LWT).

Reply:
If alternator would ship with LWT always option as default, this would become widespread.

As part of alternator tests we might unintentionally focus on this case...

Reply:
Fixed to smaller batches, but since we don't use batched insert, it does not affect tests, only data preparation